qbft/spectest/tests/prepare/imparsable_prepare_data.go

```diff
@@ -9,21 +9,18 @@ import (
 
 // ImparsableProposalData tests a prepare msg received with imparsable data
 func ImparsableProposalData() *tests.MsgProcessingSpecTest {
+	ks := testingutils.Testing4SharesSet()
```

Comment:
might not be relevant since we don't have a dedicated prepare data struct anymore